src/constants/ActionTypes.js

```diff
@@ -730,6 +743,7 @@ export default utils.deepKeyMirror({
 			REQUEST: null,
 		},
 		SET_ACTIVE_KEY: null,
+		SET_ACTIVE_KEYS: null,
```

Comment:
Does it make sense to set multiple views as active?

Reply:
no, fixed